src/main/java/domain/card/Rank.java

```diff
@@ -0,0 +1,34 @@
+package domain.card;
+
+public enum Rank {
+
+    KING(Score.valueOf(10), "K"),
+    QUEEN(Score.valueOf(10), "Q"),
+    JACK(Score.valueOf(10), "J"),
+    TEN(Score.valueOf(10), "10"),
+    NINE(Score.valueOf(9), "9"),
+    EIGHT(Score.valueOf(8), "8"),
+    SEVEN(Score.valueOf(7), "7"),
+    SIX(Score.valueOf(6), "6"),
+    FIVE(Score.valueOf(5), "5"),
+    FOUR(Score.valueOf(4), "4"),
+    THREE(Score.valueOf(3), "3"),
+    TWO(Score.valueOf(2), "2"),
+    ACE(Score.valueOf(1), "A");
```

Comment:
```suggestion
    THREE(Score.valueOf(3), "3"),
    TWO(Score.valueOf(2), "2"),
    ACE(Score.valueOf(1), "A"),
    ;
```

다음 enum이 추가될 때를 위해 다음과 같이 두면 어떨까요?

Reply:
이렇게도 작성이 가능하군요!
하나 배워갑니다 😆 감사합니다!